docs/src/index.md

```diff
@@ -15,6 +15,11 @@ ParameterDistributions.jl                   | Structures to hold prior and poste
 DataContainers.jl                           | Structure to hold model parameters and outputs
 Localizers.jl                               | Covariance localization kernels
 
+## Learning the amplitude and vertical shift of a sine curve
+![Ensemble of parameter estimates by iteration](assets/sinusoid_example_params.gif)![Ensemble of resulting curves by iteration](assets/sinusoid_example_curves.gif)
```

Comment:
is there a way of putting these side-by-side? Also, the two plots aren't in sync for me.

Reply:
Hmm, I am not having the sync problem, and I don't know if there's a way to make them sync. I think the easiest way may be to combine them into a single gif. I will do that.